packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,522 @@
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView selectionMode="multiple" disabledKeys={[3]} />
+```
+
+### Card actions
+A common use case is to allow a user to perform actions on one or more cards. The example below illustrates how one could use ActionBar to trigger an action on the
+currently selected card.
+
+```tsx example
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+import Copy from '@spectrum-icons/workflow/Copy';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Edit from '@spectrum-icons/workflow/Edit';
+
+let items = [
+  // Same items as in the pokemon examples...
+  ///- begin collapse -///
+  {id: 1, name: 'Bulbasaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png'},
+  {id: 2, name: 'Ivysaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/002.png'},
+  {id: 3, name: 'Venusaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png'},
+  {id: 4, name: 'Charmander', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png'},
+  {id: 5, name: 'Charmeleon', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png'},
+  {id: 6, name: 'Charizard', type: 'Fire, Flying', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png'}
+  ///- end collapse -///
+];
+
+function CardViewActions() {
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <ActionBarContainer height="500px">
+      <CardView items={items} selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="500px" maxWidth="800px" layout={GridLayout} aria-label="CardView Pokemon with ActionBar example" selectionMode="single">
+        {item => (
+          <Card textValue={item.name} width={item.width} height={item.height}>
+            <Image src={item.src} />
+            <Heading>{item.name}</Heading>
+            <Text slot="detail">PNG</Text>
+            <Content>{item.type}</Content>
+          </Card>
+        )}
+      </CardView>
+      <ActionBar
+        selectedItemCount={selectedKeys.size}
+        onClearSelection={() => {
+          setSelectedKeys(new Set());
+        }}
+        onAction={key => alert(`Performing ${key} action on ${selectedKeys.keys().next().value}`)}>
+        <Item key="edit">
+          <Edit />
+          <Text>Edit</Text>
+        </Item>
+        <Item key="copy">
+          <Copy />
+          <Text>Copy</Text>
+        </Item>
+        <Item key="delete">
+          <Delete />
+          <Text>Delete</Text>
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<CardViewActions />
```

Comment:
Noticed that shift tabbing from the ActionBar doesn't go back to the card view for some reason...

Reply:
Hrm, it seems to work sometimes but the other times focus moves to the "card action" anchor element even though the previously focused card in the card view still has tab index of 0. Will need to investigate, can't reproduce in the storybook

Reply:
Did some more digging, the culprit is FocusScope. When we select a card and tab into the ActionBar when it appears, the card is registered as the `nodeToRestore`. When we then shift+tab, it is also the `nextNode` since it is the next focusable node after the ActionBar's currently focused button and thus hits this if statement in `onKeyDown` from `useRestoreFocus`: https://github.com/adobe/react-spectrum/blob/7e8a7f69685139b3574453b424a5ad4c8fa4e808/packages/%40react-aria/focus/src/FocusScope.tsx#L424-L426

Since the card is outside the ActionBar's FocusScope, we shift focus to the focusable node before the Card (an anchor element in the docs) and thus focus doesn't go back into the CardView. This is a bit tough to resolve since this behavior is expected for trigger buttons w/ dropdowns (e.g. MenuTrigger) where we want tab/shift tab to move focus to the element after/before the trigger button (see https://reactspectrum.blob.core.windows.net/reactspectrum/5a6d21495e5b3d322108604b8d9c5d04390f544a/storybook/index.html?path=/story/menutrigger--menu-closes-on-blur). In the case of CardView, the Card is essentially equivalent to the trigger button of the menu...